test/e2e/ps_test.go

```diff
@@ -104,11 +104,13 @@ var _ = Describe("Podman ps", func() {
 		SkipIfRemote()
 		_, ec, _ := podmanTest.RunLsContainer("")
 		Expect(ec).To(Equal(0))
+		_, ec, _ = podmanTest.RunLsContainer("")
+		Expect(ec).To(Equal(0))
```

Comment:
why the same code run twice

Reply:
Before `ps -l` would display all containers. To test for this case, we need at least two running containers.

Reply:
We need two running containers.  The bug is that --latest should only show the 1 latest containers.  The test was failing to find the bug, since there was only one running container.